react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -1,38 +1,43 @@
 import React, { useState } from 'react';
+
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
 import { faCircle } from '@fortawesome/free-regular-svg-icons';
+
 import { faPlay, faPause } from '@fortawesome/free-solid-svg-icons';
+
 export default function Stopwatch({ into }) {
-  const [begin, realTime] = useState(0);
+  const [begin, setBegin] = useState(0);
   const [timerOn, setTime] = useState(false);
   const [clearIntervalID, setIntervalID] = useState();
 
   function onStart() {
-    setTime(true);
-    const interval = setInterval(() => {
-      realTime(begin => begin + 1);
-    }, 1000);
-    setIntervalID(interval);
-  }
+    if (timerOn) {
+      clearInterval(clearIntervalID);
+      setTime(false);
+      setIntervalID(undefined);
 
-  function handlePause() {
-    clearInterval(clearIntervalID);
-    setTime(false);
+    } else {
+      setTime(true);
+      const interval = setInterval(() => {
+        setBegin(begin => begin + 1);
+      }, 1000);
+      setIntervalID(interval);
+    }
   }
+
   function resetOnPause() {
-    if (!timerOn) return realTime(0);
+    if (!timerOn) return setBegin(0);
   }
 
+  const isPlaying = !timerOn ? faPlay : faPause;
+
   return (
     <>
       <div className='play-on'>
         <h1 className='num'>{begin}</h1>
         <FontAwesomeIcon icon={faCircle} className="circle" onClick={resetOnPause} />
-        {!timerOn
-          ? <FontAwesomeIcon onClick={onStart} icon={faPlay} className='play'/>
-          : (
-            <FontAwesomeIcon icon={faPause} onClick={handlePause} className='pause'/>)
-        }
+        <FontAwesomeIcon onClick={onStart} icon={isPlaying} className='play'/>
```

Comment:
I chatted with Dylan and saw how you guys are doing this. I approve. This is better than <i>.